src/main/java/nextstep/qna/domain/Answer.java

```diff
@@ -76,4 +76,9 @@ public void toQuestion(Question question) {
     public String toString() {
         return "Answer [id=" + getId() + ", writer=" + writer + ", contents=" + contents + "]";
     }
+
+    public DeleteHistory delete() {
+        this.deleted = true;
+        return new DeleteHistory(ContentType.ANSWER, getId(), getWriter(), LocalDateTime.now());
```

Comment:
정적생성자를 사용하는건 어떨까요? 

```suggestion
        return DeleteHistory.createAnswer(getId(), getWriter(), LocalDateTime.now());
```

질문과 답변으로 생성되니 질문도 만들수 있을것 같아요 😄